consensus/fast-sync/src/create.rs

```diff
@@ -3,7 +3,9 @@ use std::{fmt::Write, fs::write};
 use clap::Parser;
 use tower::{Service, ServiceExt};
 
-use cuprate_blockchain::{config::ConfigBuilder, service::DatabaseReadHandle, RuntimeError};
+use cuprate_blockchain::{
+    config::ConfigBuilder, cuprate_database::RuntimeError, service::DatabaseReadHandle,
+};
```

Comment:
Not sure what re-export standards are, but `cuprate_blockchain` re-exports `cuprate_database` like this so the types/traits match up.

Maybe it should be `cuprate_blockchain::database`?

Reply:
It should only be `{RuntimeError, InitError}` that is needed.

Although `cuprate_blockchain::service::DatabaseReadHandle::env() -> &Arc<ConcreteEnv>` exposes the actual database, which would only be used with a bunch of the traits from `cuprate_database`.

I set this to errors only for now, can change later.

Reply:
Actually, `cuprate_blockchain` needs a few traits internally. Is there a downside to re-exporting all of `cuprate_database`?

Reply:
Just that there is extra indirection, needing to go through `cuprate_database` first.

Reply:
https://github.com/Cuprate/cuprate/pull/160/files#r1655543354